src/bundle/Resources/public/scss/fieldType/preview/_ezimageasset.scss

```diff
@@ -3,17 +3,23 @@
 
     .ez-field-preview {
         &__image-wrapper {
-            flex-basis: 50%;
-            margin-right: 2rem;
+            flex: 1 1 auto;
```

Comment:
I can see that `ezimage` and `ezimageasset` field type styles share the same CSS code. Maybe you should think of creating a special class for such situations?

Reply:
Yes, they are nearly identical, except one, first line. It also got my attention. I think that this is not necessarily in the scope of this task, but of course, we can manage this here. Do you have any idea where extract this? Do we want to share this similar code with `@extend` or in another way? ping @sunpietro @dew326 🙂